submit.py

```diff
@@ -377,8 +379,12 @@ def show_judgement(submission_url, cfg):
             success = status_id == _ACCEPTED_STATUS
             try:
                 root = fragment_fromstring(status['row_html'], create_parent=True)
-                cpu_time = root.find('.//*[@data-type="cpu"]').text
-                status_text += " (" + cpu_time + ")"
+                cpu_time = root.xpath('.//*[@data-type="cpu"]')[0].text_content()
+                try:
+                    score = re.findall('\(([\d\.]+)\)', root.xpath('.//*[@data-type="status"]')[0].text_content())[0]
```

Comment:
This line gives me this error:
```
SyntaxWarning: invalid escape sequence '\('
```

I think you want a raw-string here:
```suggestion
                    score = re.findall(r'\(([\d\.]+)\)', root.xpath('.//*[@data-type="status"]')[0].text_content())[0]
```